conftest.py

```diff
@@ -36,6 +37,9 @@
     "saleor.graphql.webhook.tests.benchmark.fixtures",
     "saleor.tax.tests.fixtures",
     "saleor.webhook.tests.subscription_webhooks.fixtures",
+] + [
+    fixture_file.replace("/", ".").replace(".py", "")
+    for fixture_file in glob("saleor/**/tests/fixtures/[!__]*.py", recursive=True)
```

Comment:
I don't love this depending on magic

Reply:
Wouldn't it be just as easy to create `fixtures/__init__.py` with `from .checkout_line import *`?